library/alloc/src/boxed.rs

```diff
@@ -589,6 +589,38 @@ impl<T> Box<[T]> {
     pub fn new_zeroed_slice(len: usize) -> Box<[mem::MaybeUninit<T>]> {
         unsafe { RawVec::with_capacity_zeroed(len).into_box(len) }
     }
+
+    /// Constructs a new boxed slice with uninitialized contents, with the memory
```

Comment:
Saying 'uninitialized contents' sounds confusing when we also state that the memory is zeroed. I think it would be better to write out `MaybeUninit<T>`, to make it clear that the allocated memory is initialized.

Reply:
I copied that from another `_zeroed` function, should I fix those too?